src/Core/Addon.php

```diff
@@ -85,6 +85,11 @@ public static function getAdminList()
 	{
 		$addons_admin = [];
 		$addons = DI::config()->get('addons');
```

Comment:
How about just this?
```suggestion
		$addons = DI::config()->get('addons') ?? [];
```

Reply:
I was afraid that the foreach loop fails because it tries to get key & values, bu you're probably right

Reply:
Problem is null values.